test/datomish/test/query.cljc

```diff
@@ -684,3 +684,95 @@
                                    "something")
                              '[[?save]]]]}
                   conn)))))
+
+(deftest-db test-find-specs-expansion conn
+  (let [attrs (<? (<initialize-with-schema conn save-schema))]
+    ;; Relation.
+    (is (= {:select (list [:fulltext_datoms0.v :title])
+            :modifiers [:distinct]
+            :from (list [:fulltext_datoms 'fulltext_datoms0])
+            :where (list :and [:= :fulltext_datoms0.a (:save/title attrs)])}
+           (expand [:find '?title
+                    :in '$
+                    :where '[?save :save/title ?title]]
+                   conn)))
+
+    ;; Tuple. We expect only one result, and indeed we only take one.
+    ;; No need for :distinct in this case!
+    (is (= {:select (list [:fulltext_datoms0.v :title])
+            :modifiers []
+            :limit 1
+            :from (list [:fulltext_datoms 'fulltext_datoms0])
+            :where (list :and [:= :fulltext_datoms0.a (:save/title attrs)])}
+           (expand [:find '[?title]
```

Comment:
Can I get a test tuple with multiple columns, please?  This is a case that I thought through with the "single column find spec" above (which doesn't apply to that case).